executor/common_linux.h

```diff
@@ -766,6 +766,138 @@ static void initialize_devlink_pci(void)
 #endif
 #endif
 
+#if SYZ_EXECUTOR || SYZ_NIC_VF
+#include <ifaddrs.h>
+#include <linux/ethtool.h>
+#include <linux/sockios.h>
+
+struct vf_intf {
+	char pass_thru_intf[IFNAMSIZ];
+	int ppid; // used by Child
+};
+
+static struct vf_intf the_vf_struct, *vfp = &the_vf_struct;
+
+static void find_vf_interface(void)
+{
+	char cmdline[256];
+	struct ifaddrs* addresses = NULL;
+	int pid = getpid();
+	int ret = 0;
+
+	memset(vfp, 0, sizeof(struct vf_intf));
+
+	if (system("lspci | grep ^00:11.0") != 0)
```

Comment:
The code in pkg/host/features_linux.go has already detected if this feature is present or not.
If it's present (flag_nic_vf is set), then nothing here should fail. If we silently ignore all errors, this will just silently degrade in future and nobody will notice.
Fail loudly with fail function on any errors. Then future breakages will  be noticed.

Reply:
As you suggest, now using flag_nic_vf check instead of checking lspci.